app/src/components/seo.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * SEO component that queries for data with
+ * Gatsby's useStaticQuery React hook
+ *
+ * See: https://www.gatsbyjs.org/docs/use-static-query/
+ */
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+import { useStaticQuery, graphql } from "gatsby"
+
+
+function SEO({ description, lang, meta, keywords, title }) {
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            description
+            author
+          }
+        }
+      }
+    `
+  )
+
+const metaDescription = description || site.siteMetadata.description
+
+  return (
+    <Helmet
```

Comment:
Helmet is a library that manages `<head>` components for you. The Gatsby template comes with it by default, and I kind of like it. See: https://github.com/nfl/react-helmet